elementary/monitor/api/tests/tests.py

```diff
@@ -433,19 +459,25 @@ def _get_total_tests_results(
     ) -> Dict[str, TotalsSchema]:
         totals: Dict[str, TotalsSchema] = dict()
         for test in test_metadatas:
+            if not test.model_unique_id:
```

Comment:
Not sure if there's something to do with it,
but this seems to be highly repeated, can / should we do it somewhere earlier in the flow?

Reply:
Same reason as above with all the "Optional"s